docs/guides/route_filtering/inbound/prefix_length.md

```diff
@@ -79,3 +79,39 @@ Configuration examples:
     add action=reject chain=ipv4-size prefix-length=0-7
     add action=reject chain=ipv4-size prefix-length=25-32
     ```
+
+=== "Bird2"
+    ```
+    function reject_small_prefixes()
```

Comment:
I don't think protocol specific functions should come without a Suffix. I know it's in line with already present examples but i propose to change that: #33

Reply:
All Bird2 examples are update to have a suffix.

Reply:
I'm not sure why you would update all examples as that conflicts with my PR?